hs_collection_resource/views.py

```diff
@@ -104,3 +111,151 @@ def update_collection_for_deleted_resources(request, shortkey, *args, **kwargs):
     finally:
         return JsonResponse(ajax_response_data)
 
+
+def update_collection_coverages(request, shortkey, *args, **kwargs):
```

Comment:
@pkdash 
the update_collection_coverages view function is just a wrapper of internal _update_collection_coverages function.  The latter has been covered in unit test.